src/sidebar/components/SearchInput.js

```diff
@@ -39,10 +99,10 @@ export default function SearchInput({ alwaysExpanded, query, onSearch }) {
   /** @param {Event} e */
   const onSubmit = e => {
     e.preventDefault();
-    if (input.current.value || prevQuery) {
+    if (input.current?.value || prevQuery) {
```

Comment:
If this were TypeScript I'd just assert non-nullness with `input.current!.value` since there is no expected case here where the input might be null when the form is submitted.